external/llvm-project/mlir/lib/Dialect/GPU/Transforms/SerializeToHsaco.cpp

```diff
@@ -120,14 +119,6 @@ class SerializeToHsacoPass
   std::unique_ptr<std::vector<char>>
   serializeISA(const std::string &isa) override;
 
-  // Overload to allow linking in device libs
-  std::unique_ptr<llvm::Module>
-  translateToLLVMIR(llvm::LLVMContext &llvmContext) override;
-
```

Comment:
Why'd these get removed?

Reply:
(or is this just a side effect of how the diff of the "Patch external files" commit got rendered)

Reply:
The patch is rendered weirdly.  The declarations for optimizeLlvm() and translateToLLVMIR() moved from the private section to the protected section, but are otherwise unchanged.  Somebody inherited them and needed the visibility.

Reply:
Somebody is us